Makefile

```diff
@@ -44,7 +44,7 @@ all: test build
 test: clean cover.out
 
 unit: kubebuilder
-	KUBEBUILDER_ASSETS=$(KUBEBUILDER_ASSETS) go test $(MOD_FLAGS) $(SPECIFIC_UNIT_TEST) -tags "json1" -v -race -count=1 ./pkg/...
```

Comment:
I removed `-v` here because it was _really_ hard to find the test failure in the log with all of the verbose output. If `-v` is here for a good reason, I can add it back.